packages/seacas/libraries/ioss/src/utest/CMakeLists.txt

```diff
@@ -29,7 +29,9 @@ TRIBITS_ADD_EXECUTABLE(
 TRIBITS_ADD_TEST(
         Utst_blob_write_read
 	NAME Utst_blob_write_read
-	NUM_MPI_PROCS 1
+	NUM_MPI_PROCS ${NPROCS}
+	COMM mpi serial
```

Comment:
@gsjaardema, note, new behavior of TriBITS as for PR #8115 is to skip a test from the test suite non-MPI (i.e. "serial") builds if `NUM_MPI_PROCS` is greater than 1.  If you want to run a 1-proc version of this test an a multi-proc version, you should add those as two separate tests, like we did with MueLu in PR #8173.  (NOTE: If there is interest in using that approach, that function could be added to TriBITS proper so SEACS could reuse it.  But if this pattern in rare in SEACAS, it is not worth it.  Just define two separate tests.)

Reply:
Actually, I think that this change was not wanted since for some reason this test fails in Trilinos builds on certain platforms, but doesn't fail in SEACAS...